x-pack/plugins/cases/public/components/all_cases/assignees_filter.tsx

```diff
@@ -104,6 +114,7 @@ const AssigneesFilterPopoverComponent: React.FC<AssigneesFilterPopoverProps> = (
           onClick={togglePopover}
           isLoading={isLoadingData}
           isSelected={isPopoverOpen}
+          numFilters={availableOwners.length}
```

Comment:
We should remove this, right? `availableOwners` is not related to the assignees.